packages/falcon-front-kit/src/Apollo/apolloStateToObject.ts

```diff
@@ -4,8 +4,9 @@
  * @param {string} key Apollo state key
  * @returns {object} Expanded object
  */
-export const apolloStateToObject = (state, key) => {
+export function apolloStateToObject(state: Object, key: string): Object {
```

Comment:
```suggestion
export function apolloStateToObject(state: object, key: string): Object {
```

shouldn't it be `object`?

Reply:
done